app/code/Magento/Swatches/Test/Mftf/Test/StorefrontCustomerCanChangeProductOptionsUsingSwatchesTest.xml

```diff
@@ -22,8 +22,9 @@
             <createData entity="Simple_US_Customer" after="createCategory" stepKey="createCustomer"/>
         </before>
         <after>
+            <!-- Logout customer -->
+            <actionGroup ref="StorefrontCustomerLogoutActionGroup" stepKey="customerLogoutStorefront"/>
```

Comment:
Please use the same step key: logoutFromCustomer

Reply:
updated.